test/helper/objects.go

```diff
@@ -40,6 +40,11 @@ func CreateClass(t *testing.T, class *models.Class) {
 	t.Helper()
 	params := schema.NewSchemaObjectsCreateParams().WithObjectClass(class)
 	resp, err := Client(t).Schema.SchemaObjectsCreate(params, nil)
+	// TODO shall be removed with the DB is idempotent
```

Comment:
nitpick: making it `TODO-RAFT` would make it easier to catch with grep once we finish up the raft implementation

Reply:
good idea